docs/troubleshooting/attach_state_snapshot.md

```diff
@@ -0,0 +1,50 @@
+---
+id: attach-state-snapshot
+title: Attach State Snapshot
+sidebar_label: Attach State Snpashot
+description: Instructions on attaching a supporting state snapshot.
+---
+
+## Terminology {#terminology}
+State Snapshot is different from DB snapshot.
+State Snapshot is checkpoint of some columns of the full DB taken at the epoch boundary.
```

Comment:
Technically it's a checkpoint of the whole db with some unneeded columns deleted. At the end of the day we have hardlinks to all of the sst files (at least without compaction). I would keep it simple and just say it's a checkpoing of the full db, and not mention about being selective of some columns. This is in line with the expected size of the snapshot too.